src/garage/sampler/utils.py

```diff
@@ -75,10 +83,16 @@ def rollout(env,
             timestep = 0.05
             time.sleep(timestep / speedup)
 
+    observations = np.array(observations)
+    next_observations = np.vstack(
```

Comment:
Does modifying the return still allow you to pass all of the tests? If so, then this is fine. if not, create a separate rollout function specifically for pearl.